modules/tinymce/src/core/main/ts/api/OptionTypes.ts

```diff
@@ -50,6 +50,8 @@ interface BaseEditorOptions {
   add_form_submit_trigger?: boolean;
   add_unload_trigger?: boolean;
   allow_conditional_comments?: boolean;
+  allow_extra_mathml_attributes?: string[];
+  allow_extra_mathml_elements?: string[];
```

Comment:
I don't think word `extra` fits with our other configuration options. Some suggestions

- `allow_mathml_attributes` and `allow_mathml_elements` would fit better with the other `allow` configs
- `extended_mathml_attributes` and `extended_mathml_elements` is perhaps the best match, in the style of `extended_valid_elements`

Reply:
It need the extra or something similar since it doesn't replace the whole check it adds additional things on top of what DOMPurify already allows. Otherwise they would need to pass in the whole spec in these options.
The original option name was this but there where complaints about that internally of being confusing and I agree with that.

Reply:
Yes let go with that. It is a bit different to the extended_valid_elements since that allows you to define both the element and the attributes. But docs should clear that up. Option names that make sense is hard.